### PR TITLE
fix test collection sort order

### PR DESCRIPTION
**Pull request by bernt-matthias**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by mvdbeek — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

File: tools/mothur/chimera.check.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@ echo 'chimera.check(
         <expand macro="logfile-output"/>
         <data name="chimeracheck.chimeras" format="txt" from_work_dir="fasta.chimeracheck.chimeras" label="${tool.name} on ${on_string}: chimeracheck.chimeras"/>
         <collection name="images" type="list" label="${tool.name} on ${on_string}: SVG images">
-            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.chimeracheck\.svg"/>
+            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.chimeracheck\.svg"/ ftype="svg">
```
Comment (mvdbeek) [inline tools/mothur/chimera.check.xml:70]:
```suggestion
            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.chimeracheck\.svg"/ ftype="svg">
```
```suggestion
            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.chimeracheck\.svg" ftype="svg">
```

**Comment by bernt-matthias** [comment]
Thanks @mvdbeek . I guess we should leave the PR open until the problem described in https://github.com/galaxyproject/galaxy/pull/10304 is fixed?

**Comment by mvdbeek** [comment]
If this passes tests I don't see why we shouldn't merge ?

**Comment by bernt-matthias** [comment]
Even if we know that the test is 'wrong', i.e. incomplete.

**Comment by mvdbeek** [comment]
That's fine, at least some tests are more correct.

**Comment by bernt-matthias** [comment]
I think we can close this now, since the sort order is only enforced for newer profiles.

**Pull request closed without merging**